dftimewolf/lib/processors/gcp_logging_timesketch.py

```diff
@@ -262,23 +260,22 @@ def _ProcessLogContainer(self, logs_container: containers.GCPLogs) -> None:
     with open(logs_container.path, 'r') as input_file:
       for line in input_file:
         transformed_line = self._ProcessLogLine(
-            line, logs_container.filter_expression, logs_container.project_name)
+            line, logs_container.name)
         if transformed_line:
           output_file.write(transformed_line)
           output_file.write('\n')
     output_file.close()
 
-    timeline_name = 'GCP logs {0:s} "{1:s}"'.format(
-        logs_container.project_name, logs_container.filter_expression)
+    current_timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S')
+    timeline_name = f'{current_timestamp}'
```

Comment:
Perhaps timeline name should be a bit more descriptive than just the timestamp?

Reply:
Updated to use `project_id` in timeline name i.e. `<project_id>_<yyyymmddhhmmss>`.